roles/pulp/README.md

```diff
@@ -48,6 +48,7 @@ Role Variables:
     dictionary are variable names, and the values can be nested. Please see [pulpcore configuration
     docs](https://docs.pulpproject.org/en/3.0/nightly/installation/configuration.html#id2) for
     documentation on the possible values.
+  * `pulp_settings.content_origin`: **Required**. The URL to the pulp-content host that clients will access, and that will be appended to in HTTP responses by multiple content plugins. Any load balancers / proxies (such as those in the `pulp-webserver` role) normally should be specified instead of the pulp content host itself. Syntax is `([http|https)://(hostname|ip)[:port]`.
```

Comment:
The text looks great.

I think maybe you're missing a `]` after `https` though.

Also can you line wrap it like the other lines (not rely on the viewer to line-wrap)?

Reply:
I'll wrap it.

I think the [ at the beginning should be removed. Since in manpages, "[]" denote optional parameters.